### PR TITLE
Fix compile warnings

### PR DESCRIPTION
**Pull request by pali**

### CODE REVIEW

File: test.c [hunk 1]
```diff
@@ -251,12 +252,14 @@ int main()
...
     }
     else
-        printf("SUCCESS\tGot symbol from global handle: %p\n", fwrite_local);
-    char * hello_world = "Hello world from local fwrite!\n";
+        printf("SUCCESS\tGot symbol from global handle: %p\n", *(void **)&fwrite_local);
+    {
+    const char *hello_world = "Hello world from local fwrite!\n";
     fwrite_local(hello_world,sizeof(char),strlen(hello_world),stderr);
     fflush(stderr);
+    }
```
Comment (traversaro) [inline test.c]: I am not sure what convention we follow in the rest of the repo, but just to understand, in this case you did not indented intentionally as the scope is not associated with a `if`/`for`/`while`?

Comment (pali) [reply test.c]: Should I indent those 3 lines in that block by another level?

Comment (traversaro) [reply test.c]: This is the first scope of this kind in the codebase, but I briefly checked with https://zed0.co.uk/clang-format-configurator/ and all major styles indent lines inside scopes like this, so I think it make sense to indent those lines, thanks!

Comment (pali) [reply test.c]: I separated it in next commit (with explained commit message) and fixed it there.

**Comment by pali** [comment]
Hi! I changed pull request as you wanted. Could you please look at it?

**Comment by traversaro** [comment]
Thanks a lot @pali !

**Pull request merged**